spring-jdbc/spring-jdbc.gradle

```diff
@@ -5,6 +5,7 @@ dependencies {
 	compile(project(":spring-core"))
 	compile(project(":spring-tx"))
 	optional(project(":spring-context"))  // for JndiDataSourceLookup
+	optional("org.springframework.data:spring-data-commons:2.1.5.RELEASE")
```

Comment:
This isn't going to work as we end up with a dependency cycle. Spring Data depends on Spring Framework, so Spring Framework cannot depend on Spring Data.

Reply:
Hi, @mp911de, thanks for the review. I wasn't aware of the dependency problem, as Data-Commons isn't dependent on spring-jdbc directly. 

Could you please review the main class of this PR, and tell if it is even worth considering? Would this work if we transfer it to one of the Spring Data modules?